File: src/test/java/fi/metatavu/metaform/test/functional/builder/FilledAccessTokenProvider.java
package fi.metatavu.metaform.test.functional.builder;

import fi.metatavu.jaxrs.test.functional.builder.auth.AccessTokenProvider;

import java.io.IOException;

/**
 * Access token provider with prefilled token
 */
public class FilledAccessTokenProvider implements AccessTokenProvider {

  private final String token;

  public FilledAccessTokenProvider(String token) {
    this.token = token;
  }

  @Override
  public String getAccessToken() {
    return token;
  }
}

File: src/test/java/fi/metatavu/metaform/server/test/functional/builder/FilledAccessTokenProvider.kt
package fi.metatavu.metaform.server.test.functional.builder

import fi.metatavu.jaxrs.test.functional.builder.auth.AccessTokenProvider

/**
 * Access token provider with prefilled token
 */
class FilledAccessTokenProvider(private val token: String) : AccessTokenProvider {
    override fun getAccessToken(): String {
        return token
    }
}